### PR TITLE
Reduce stress by sleeping for random interval

### PR DESCRIPTION
**Pull request by prsurve**

### CODE REVIEW

File: rdr/busybox/scripts/kernel_untar.sh [hunk 1]
```diff
@@ -69,6 +69,21 @@ set_running_state()
     sync -f $MOUNT
 }
 
+# Run kernal untar
+untar_kernal_file()
+{
+    # Sleeping for some random interval of time to reduce stress
+    echo "Sleeping for $tm"
+    sleep $tm
```
Comment (shylesh) [inline rdr/busybox/scripts/kernel_untar.sh:77]: @prsurve where do you get this $tm from ?

File: rdr/busybox/scripts/kernel_untar.sh [hunk 2]
```diff
@@ -104,6 +119,7 @@ if [ -f "$MOUNT""/""$MASTER_COPY" ]; then
 fi
 
 # From here keep a loop of untaring and renaming the kernel dirs
+sleep_time=$(shuf -i 180-500 -n1)
```
Comment (shylesh) [inline rdr/busybox/scripts/kernel_untar.sh:122]: where are we using this sleep_time?

**Review by keesturam — APPROVED** [review]

File: rdr/busybox/scripts/kernel_untar.sh [hunk 3]
```diff
@@ -69,6 +69,21 @@ set_running_state()
     sync -f $MOUNT
 }
 
+# Run kernal untar
+untar_kernal_file()
+{
+    # Sleeping for some random interval of time to reduce stress
+    echo "Sleeping for $sleep_time"
+    sleep $sleep_time
```
Comment (shylesh) [inline rdr/busybox/scripts/kernel_untar.sh:77]: @prsurve , do we want to use same time every iteration? OR do we want to change sleep time every iteration?

**Review by shylesh — DISMISSED** [review]
Lgtm

**Review by sidhant-agrawal — APPROVED** [review]
lgtm

**Review by am-agrawa — APPROVED** [review]

**Pull request merged**